_maps/shuttles/emergency_pubby.dmm

```diff
@@ -141,493 +133,832 @@
 /turf/open/floor/plating,
 /area/shuttle/escape)
 "az" = (
-/obj/structure/reagent_dispensers/fueltank,
-/turf/open/floor/mineral/titanium/yellow,
+/obj/structure/chair/comfy{
+	dir = 1
+	},
+/obj/structure/window/reinforced,
+/turf/open/floor/plasteel/darkgreen/side{
+	tag = "icon-darkgreen (SOUTHWEST)";
+	icon_state = "darkgreen";
+	dir = 10
+	},
 /area/shuttle/escape)
 "aA" = (
-/obj/structure/sign/nanotrasen,
-/turf/closed/wall/mineral/titanium,
+/obj/structure/chair/comfy{
+	dir = 1
+	},
+/obj/structure/window/reinforced,
+/turf/open/floor/plasteel/darkgreen/side,
 /area/shuttle/escape)
 "aB" = (
-/obj/machinery/door/airlock/public/glass{
-	name = "Emergency Shuttle Cockpit";
-	req_access_txt = "19"
+/obj/structure/window/reinforced,
+/obj/item/twohanded/required/kirbyplants{
+	icon_state = "plant-11"
+	},
+/turf/open/floor/plasteel/darkgreen/side{
+	tag = "icon-darkgreen (SOUTHEAST)";
+	icon_state = "darkgreen";
+	dir = 6
 	},
-/turf/open/floor/mineral/titanium/blue,
 /area/shuttle/escape)
 "aC" = (
-/obj/structure/chair{
+/obj/structure/chair/comfy{
 	dir = 1
 	},
-/turf/open/floor/mineral/plastitanium/brig,
+/obj/structure/window/reinforced,
+/turf/open/floor/plasteel/darkgreen/side{
+	tag = "icon-darkgreen (SOUTHEAST)";
+	icon_state = "darkgreen";
+	dir = 6
+	},
 /area/shuttle/escape)
 "aD" = (
-/obj/machinery/door/airlock/titanium{
-	name = "Emergency Shuttle Airlock";
-	req_access_txt = "2"
+/obj/structure/chair/comfy{
+	dir = 1
+	},
+/turf/open/floor/plasteel/darkgreen/side{
+	tag = "icon-darkgreen (SOUTHWEST)";
```

Comment:
Anything that's tag = "icon is a dv

I'd mark them all but I'm on my phone till next week.